e2e/tests/integration/arrivals-filter.ts

```diff
@@ -34,4 +34,93 @@ describe('Arrivals page filter', () => {
       .get('.danger > :nth-child(1)').contains("TS0123")
 
   });
+
+ //valid service types for filter ("J", "S", "Q", "G", "B", "R", "C", "L")
+
+  it('Filter flight by passenger flight only for ACL Forecast when LoadFactor is zero and valid service type', () => {
+    cy
+      .addTestFlight(
+        {
+          "Status" : "ACL Forecast",
+          "SchDT": todayAtUtcString(0, 55),
+          "EstDT": todayAtUtcString(1, 5),
+          "EstChoxDT": todayAtUtcString(16, 11),
+          "ActDT": todayAtUtcString(16, 7),
+          "ActChoxDT": todayAtUtcString(16, 45),
+          "ActPax": 300,
+          "ServiceType" : "J",
+          "LoadFactor" : 0
+        }
+      )
+      .asABorderForceOfficer()
+      .waitForFlightToAppear("TS0123")
+      .get("#toggle-arrival").click()
+      .get('#arrivals > div').contains("No flights to display");
+
+  });
+
+  it('Not Filter flight by passenger flight for ACL Forecast when LoadFactor is not zero and valid service type', () => {
```

Comment:
we can probably cover most of these cases with unit tests rather than E2E tests which are slow.